cli/tools/bench.rs

```diff
@@ -123,6 +132,74 @@ pub trait BenchReporter {
   fn report_result(&mut self, desc: &BenchDescription, result: &BenchResult);
 }
 
+#[derive(Debug, Serialize)]
+struct JsonReporterResult {
```

Comment:
**Bikeshedding:** I would perhaps split up the formatting a bit; now each result field includes the runtime and CPU information, and groups are only an entry in the result data and not actually _groups_ as the name might suggest.

I might rather go with something like:
```json
{
  "runtime": "...",
  "cpu": "...",
  "groups": [
    {
      "name": null,
      "results": [
        {
          "name": "nop()",
          "origin": "file.js",
          "baseline": false,
          "result": { "ok": { } }
        }
      ]
    }
  ]
}
```